swift_browser_ui_frontend/build_wasm.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT="$(realpath $0)"
+SCRIPT_ROOT=$(dirname "$SCRIPT")
+
+WASM_ROOT="${SCRIPT_ROOT}"/wasm/
+
```

Comment:
i think we need to first do a `docker` command check that the command is available and one can run it e.g. by testing `docker version`.

Reply:
Added checks for docker